pajbot/modules/basic/emotes.py

```diff
@@ -144,7 +144,7 @@ def do_print(source, **rest):
             self.print_emotes(source, manager)
 
         if self.settings[f"custom_{manager.friendly_name.lower()}_response"]:
```

Comment:
```suggestion
        if self.settings[f"custom_{manager.friendly_name.lower()}_response"] != "":
```

Reply:
The idea of this is to specifically match nothing. Is this still applicable?

Reply:
nevermind, I misunderstoon